content/applications/inventory_and_mrp/manufacturing/workflows/allocation.rst

```diff
@@ -0,0 +1,140 @@
+==================
+Allocation reports
+==================
+
+.. |SO| replace:: :abbr:`SO (sales order)`
+.. |SOs| replace:: :abbr:`SOs (sales orders)`
+.. |MO| replace:: :abbr:`MO (manufacturing order)`
+.. |MOs| replace:: :abbr:`MOs (manufacturing orders)`
+.. |RfQ| replace:: :abbr:`RfQ (request for quotation)`
+
+When fulfilling sales orders (SOs), or sourcing components for manufacturing orders (MOs), it is
+sometimes necessary to prioritize one |SO| or |MO| over another. In situations where there is
+insufficient stock on-hand to fulfill every |SO| or |MO|, this means ensuring that products and
+components are reserved for those that must be prioritized.
+
+In Odoo *Manufacturing*, allocation reports are used on |MOs| to assign products to specific sales
+orders |SOs|, or components to specific |MOs|. This ensure that the products or components are
+available for those orders, and are not used by mistake.
+
+Configuration
+=============
+
+To use allocation reports, the *Allocation Report for Manufacturing Orders* feature must be enabled.
+To do so, navigate to :menuselection:`Manufacturing app --> Configuration --> Settings`, and tick
+the checkbox next to :guilabel:`Allocation Report for Manufacturing Orders`.
+
+.. image:: allocation/allocation-setting.png
+   :align: center
+   :alt: The Allocation Report for Manufacturing Orders setting.
+
+For products that are sold, it is also necessary to configure them so they can be included in |SOs|.
+To do so, begin by navigating to :menuselection:`Inventory --> Products --> Products`, and select a
```

Comment:
```suggestion
To do so, begin by navigating to :menuselection:`Inventory app --> Products --> Products`, and select a
```

Reply:
Could this be done directly through the Manufacturing app? Could be a nice way to save some time, and make everything complete in one app -- but that's just a suggestion. Totally your call.

Reply:
It can be, but I prefer to always have inventory actions conducted through the Inventory app.